clients/csharp-dotnet-core/voice-assistant-test/tool/MainService.cs

```diff
@@ -308,6 +315,28 @@ private static async Task<bool> ProcessTestFiles(AppSettings appSettings)
                             testPass = false;
                         }
 
+                        if (turn.WAVFile != null)
+                        {
+                            if (!string.IsNullOrEmpty(turn.WAVFile))
+                            {
+                                if (turn.WAVFile.Contains(",", StringComparison.OrdinalIgnoreCase))
+                                {
+                                    if (turnResult.WAVFile.Split(",").Length == 2)
+                                    {
+                                        string[] arr = turnResult.WAVFile.Split(",");
+                                        arr[1] = botConnector.LengthOfSpeechInWavFile.ToString(CultureInfo.CurrentCulture);
+                                        turnResult.WAVFile = arr[0] + ", " + arr[1];
+                                    }

```

Comment:
missing else {} statement for this if {}. Should error out if there are more than one comma (test it out)

Reply:
Done